commands/scanandfixrepos.go

```diff
@@ -68,3 +85,57 @@ func (saf *ScanAndFixRepositories) downloadAndRunScanAndFix(repository *utils.Fr
 	cfp := CreateFixPullRequestsCmd{dryRun: saf.dryRun, dryRunRepoPath: filepath.Join(saf.dryRunRepoPath, repository.RepoName)}
 	return cfp.scanAndFixRepository(repository, branch, client)
 }
+
+func (saf ScanAndFixRepositories) setCommitBuildStatus(client vcsclient.VcsClient, repoConfig *utils.FrogbotRepoConfig, state vcsclient.CommitStatus, commitHash string, description string) error {
+	background := context.Background()
+	err := client.SetCommitStatus(background, state, repoConfig.RepoOwner, repoConfig.RepoName, commitHash, utils.FrogbotCreatorName, description, utils.CommitStatusDetailsUrl)
+	if err != nil {
+		log.Error("Failed to mark last commit as checked")
+		return err
+	}
+	log.Info(fmt.Sprintf("Successfully marked commit %s as checked by FrogBot", commitHash))
```

Comment:
FrogBot --> Frogbot
Also `fmt.Sprintf` isn't needed in this case, and -
```
log.Info(fmt.Sprintf("Successfully marked commit %s as checked by FrogBot", commitHash))
```
can be simplified to - 
```
log.Info("Successfully marked commit %s as scanned", commitHash)
```